cmd/crawl.go

```
@@ -153,7 +153,7 @@ func init() {
 	CrawlCmd.Flags().BoolVar(&showOutput, "show", false, "Show the output of a collect run")
 	CrawlCmd.Flags().VarP(&crawlOutputFormat, "format", "F", "Set the output format (json|yaml)")
 
-	CrawlCmd.RegisterFlagCompletionFunc("format", completionFormatData)
+	_ = CrawlCmd.RegisterFlagCompletionFunc("format", completionFormatData)
```

See comment above regarding ignoring errors.